### DISCUSSION TITLE
Why the name Uvicorn?

### DISCUSSION THREAD

**Discussion by HyperEpsilon** [Kludex/uvicorn#1124] [Q&A] [answered]
Like, it's a super cool name, but why?

**Comment by HarrySky** [comment, marked as answer]
I guess uvicorn is called uvicorn, because there is a naming scheme `*corn` used by popular Python WSGI/ASGI servers (gunicorn, hypercorn, etc) and uvicorn also uses `uvloop` (it became optional only after some time)

**Comment by nathanbrenton** [comment]
v is the lower-case symbol for Greek letter "nu" which designates the "n" sound. Given the unicorn logo and that uvloop was/is an add-on, I wonder how relevant the Greek-letter angle was considered when coming up with the name.
